### PR TITLE
Document references to database by name instead of path

### PR DESCRIPTION
**Pull request by benbc**

### CODE REVIEW

**Comment by benbc** [comment]
@jjaderberg or @mariascharin: Please could you review these docs changes?

File: manual/contents/src/operations/installation-deployment/file-locations.asciidoc [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
-|OS X desktop          |_$\{HOME\}/Documents/Neo4j/neo4j.conf_
+|Package               |Configuration                                  |Data
+|Linux or OS X tarball |_<neo4j-root>/conf/neo4j.conf_                 |_<neo4j-root>/data_
+|Windows zip           |_<neo4j-root>\conf\neo4j.conf_                 |_<neo4j-root>\data_
+|Debian/Ubuntu .deb    |_/etc/neo4j/neo4j.conf_                        |_/var/lib/neo4j_
+|Windows desktop       |_%APPDATA%\Neo4j Community Edition\neo4j.conf_ |_%APPDATA%\Neo4j Community Edition_
+|OS X desktop          |_$\{HOME\}/Documents/Neo4j/neo4j.conf_         |_$\{HOME\}/Documents/Neo4j_
 |===
+
+Please note that the structure directories and files in the data directory is internal to Neo4j and subject to change between versions without notice.
```
Comment (jjaderberg) [inline manual/contents/src/operations/installation-deployment/file-locations.asciidoc:17]: How about "Please note that the data directory is internal to Neo4j and its structure subject to change between versions without notice."

File: manual/contents/src/operations/installation-deployment/file-locations.asciidoc [hunk 2]
```diff
@@ -1,14 +1,17 @@
 [[file-locations]]
 = File Locations
 
-This table shows where important files can be found in various Neo4j distribution packages.
+This table shows where important files can be found by default in various Neo4j distribution packages.
+Many of these paths are configurable; see <<configuration-settings>> for details.
```
Comment (jjaderberg) [inline manual/contents/src/operations/installation-deployment/file-locations.asciidoc:5]: Prefer "Some" over "Many".

**Comment by jjaderberg** [comment]
Looks good! This PR seems to have a dependency on https://github.com/neo4j/neo4j/pull/6628, which is marked `NOT READY FOR MERGE`. Will that PR be ready for merge soon?

**Pull request merged**